core/sql/executor/ExHbaseIUD.cpp

```diff
@@ -1158,7 +1158,7 @@ ExHbaseAccessBulkLoadPrepSQTcb::ExHbaseAccessBulkLoadPrepSQTcb(
                       "traf_upsert_err",
                       fileNum,
                       loggingFileName_);
-   LoggingFileCreated_ = FALSE;
+   loggingFileCreated_ = FALSE;
```

Comment:
Is this line necessary now? loggingFileCreated_ belongs to the base class now I gather. Its constructor would have done this already.

Reply:
Yes. I will confirm and remove it